js/app-super-resolution/instructions.md

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Microsoft Corporation.
+
+// Licensed under the MIT license.
+
+## Steps of building project from scratch
+
+0. Prerequisites
+
+   1. Install Node.js
+
+   2. Install Java (OpenJDK).
+
+      - Download and install [OpenJDK Version 11](https://adoptopenjdk.net/). Extract into the `program files` folder in your local disk.
```

Comment:
extracting to any place should work (as long as the next step of setting JAVA_HOME is doing correctly).

Reply:
Oh, I did not know that. Thanks 👍